### ISSUE TITLE
More helpful response from the base /api/v1/ HTTP API endpoint

### ISSUE THREAD

**Issue by ttmc** [bigchaindb/bigchaindb#906]
This is something to consider for the HTTP API design. It's something I've encountered when I wanted to do simple tests of the API (i.e. without first constructing a transaction).

If I send a request to http://my-bigchaindb-server.io:9984 I get back a nice response like:
```JSON
{
  "keyring": [
    "2RTL8y4FcVe1kZobdmreSiabNBLDGXSn4FGHbMYtujVa", 
    "CcNi32PheZNS6WSSM4nKAPvrXx2bd25JQA7LxKj2xtaT"
  ], 
  "public_key": "ESYhpYZas8wr6sJ5uxuNaA1WTsSDjmZWuuvD1nkF285F", 
  "software": "BigchainDB", 
  "version": "0.8.0"
}
```

but if I send a request to http://my-bigchaindb-server.io:9984/api/v1/ I get back a response like:
```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.</p>
```

It would be nice if that response was more helpful. Maybe it could say the name of the API, e.g. BigchainDB HTTP API version x? Maybe give a link to the API documentation? Maybe there are some best practices for this?

**Comment by sbellem** [comment]
if we wish to make the API "discoverable" that is important

**Comment by diminator** [comment]
We could use the hyperlinked approach, displaying the root endpoints their

**Comment by libscott** [comment]
Fixed by #1024
